backend/services/prisonerProfileService.ts

```diff
@@ -172,9 +172,9 @@ export default ({
       showPathfinderReferButton: Boolean(!pathfinderDetails && isPathfinderReadWriteUser),
       pathfinderReferUrl: pathfinderUrl && `${pathfinderUrl}refer/offender/${offenderNo}`,
       canViewSocLink: socEnabled && canViewSocLink,
-      socProfileUrl: socEnabled && socUrl && socDetails && `${socUrl}/nominal/${String((socDetails as any).id)}`,
```

Comment:
this looks a bit naff - we normally try to exclude `/` from the config url so that the urls look nicer

Reply:
i went with the majority to keep it consistent

Reply:
you are right - i meant to take that out